vllm/model_executor/models/phi4flash.py

```diff
@@ -364,11 +405,11 @@ def forward(self,
         time_proj_bias = (self.dt_proj.bias.float() if hasattr(
             self.dt_proj, "bias") else None)
 
-        if attn_metadata.query_start_loc is not None \
-            and attn_metadata.context_lens_tensor is not None:
+        if (attn_metadata.query_start_loc is not None
+                and attn_metadata.context_lens_tensor is not None):
             scan_outputs = selective_scan_fn(
                 hidden_states,
-                mamba_cache_params.ssm_state,
+                attn_metadata.ssm_state,
```

Comment:
Same here - the attention metadata shoud not contain the ssm state (in fact, looking at the definition of the class below, it does not). I don't understand how this code works at all?!